stories/core/Typography/Label.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { Input, Label, LabelProps, Text } from '../../../src/core';
+
+export default {
+  component: Label,
+  args: {
+    children: 'Name',
+    displayStyle: 'block',
```

Comment:
Is it possible to have stories where switching `displayStyle` changes the layout? Because now the control for it is useless.

Reply:
It shows the actual behavior tho, i.e. it's not enough by itself.

I could toggle flex-direction based on arg but I think that adds too much noise